templates/registration/logged_out.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p>Logged out!</p>
+  <a href="{% url 'login'%}">Click here to login again.</a>
+
+
+{% endblock %}
```

Comment:
Not fixed

Reply:
you mean i need to add new lines after {% endblock %}?